### ISSUE TITLE
Problems with dependency to fabfuel/prophiler

### ISSUE THREAD

**Issue by Musiksammler** [bitExpert/prophiler-psr7-middleware#11]
Fabfuel's prophiler has a strict dependency to PHP 5.x so it's not possible to install it under PHP7.

He offers the branch "dev-feature/php7" to do so and that's works if installed alone.

Your middleware has a dependecy for prophiler version 1.4/1.5. When I defined "dev-feature/php7" for prophiler in my composer.json prophiler-psr7-middleware fails with the message

```
bitexpert/prophiler-psr7-middleware dev-master requires fabfuel/prophiler ^1.4 -> satisfiable by fabfuel/prophiler[1.4.0, 1.5.0] but these conflict with your requirements or minimum-stability.
```

Of course I can use the option "--ignore-platform-reqs" for the standard prophiler package but I would prefer a solution without that.

**Comment by shochdoerfer** [comment]
Good catch. I was hoping that PHP7 support would be merged much quicker in fabfuel/prophiler. I believe we can add a simple rule to the composer.json file to fix it. Just need to figure out a solution for the Travis build which will fail then (due to the locked dependencies in the composer.lock file).

**Comment by shochdoerfer** [comment]
@Musiksammler I just released version 0.3.0 to address this issue. You can now define the dependencies in your root composer.json like this to install the PHP 7 version of fabfuel/prophiler:

``` js
{
  "require": {
    "php": "^7.0",
    "fabfuel/prophiler": "dev-feature/php7",
    "bitexpert/prophiler-psr7-middleware": "^0.3.0"
  }  
}
```

**Comment by Musiksammler** [comment]
Works, thank you!
